flight/Modules/VtolPathFollower/vtol_follower_control.c

```diff
@@ -560,12 +576,29 @@ bool vtol_follower_control_loiter(float dT, float *hold_pos, float *att_adj) {
 		cmd.Pitch
 	};
 
+	const float CMD_THRESHOLD = 0.2f;
+	const float CMD_ALT_THRESHOLD = 0.4f;
```

Comment:
I think it would make sense to use the same deadbands as for `AltitudeHold`.. can we just use `AltitudeHoldSettings` here? IMO would be best to have the same AH behaviour in normal AH and loiter.